cli/utils.go

```diff
@@ -21,18 +22,34 @@ var (
 	Offset uint
 )
 
-func flush(i interface{}) {
-	fmt.Printf("%s", color.BlueString(spew.Sdump(i)))
+func logJSON(i interface{}) {
+	m, err := json.Marshal(i)
+	if err != nil {
+		logError(err)
+		return
+	}
+
+	pj, err := prettyjson.Format([]byte(m))
+	if err != nil {
+		logError(err)
+		return
+	}
+
+	fmt.Printf("\n%s\n\n", string(pj))
 }
 
 func logUsage(u string) {
-	fmt.Printf(color.YellowString("Usage:  %s\n"), u)
+	fmt.Printf(color.YellowString("\nusage: %s\n\n"), u)
 }
 
 func logError(err error) {
-	fmt.Printf("%s\n", color.RedString(err.Error()))
+	fmt.Printf("\n%s\n\n", color.RedString(err.Error()))
 }
 
 func logOK() {
```

Comment:
Maybe we can join `logOK` and `logCreated` to single variadic `logSuccess(s ...string)` function. Something like:
```go
func logSuccess(s ...string) {
	switch len(s) {
	case 1:
		fmt.Printf(..., s[0])
	case 0:
		fmt.Println("OK")
	}
}

```
It can be used as `logSuccess()` or `logSuccess("message")`.